Week-5/calculator/calculator.test.js

```diff
@@ -0,0 +1,13 @@
+const operations = require("./calculator");
+
+describe("calculator tests", () => {
+  test("Adding 1+2 should return 3", () => {
+    expect(operations.sum(1, 2)).toBe(3);
+  });
+  test("Diff 100 -50 should return 50", () => {
+    expect(operations.diff(100, 50)).toBe(50);
```

Comment:
Good

Reply:
Try to explore more variety of test cases that covers mocking functions and callbacks.